Library/src/Web/POCOWebClient/POCOWebClient.cpp

```diff
@@ -478,9 +479,17 @@ void POCOWebClient::SetFileUploadContentFromFile(HttpPayload* Payload,
 												 const char* Version,
 												 const csp::common::String& MediaType)
 {
-	/// @note this must be newed as it is deleted by the HTMLForm
-	Poco::Net::FilePartSource* Source = new Poco::Net::FilePartSource(FilePath, MediaType.c_str());
-	SetFileUploadContent(Payload, Source, Version);
+	Poco::File* File = new Poco::File(FilePath);
+	if (File->exists())
+	{
+		/// @note this must be newed as it is deleted by the HTMLForm
+		Poco::Net::FilePartSource* Source = new Poco::Net::FilePartSource(FilePath, MediaType.c_str());
+		SetFileUploadContent(Payload, Source, Version);
+	}
+	else
+	{
+		CSP_LOG_WARN_MSG("File not found.");
```

Comment:
Could we also print the file path to the log here? That'd be really useful for developers diagnosing problems in their code